blogs/2020/07/27/containers-edu.md

```diff
@@ -0,0 +1,217 @@
+---
+Order: 60
+TOCTitle: Development Containers in Education
+PageTitle: Development Containers in Education
```

Comment:
For the PageTitle and MetaDescription, can you add "... with Visual Studio Code"

Reply:
Definitely! Added to both.